clang/lib/CodeGen/CodeGenAction.cpp

```diff
@@ -980,20 +981,67 @@ CodeGenAction::CreateASTConsumer(CompilerInstance &CI, StringRef InFile) {
         return nullptr;
       }
 
-      Expected<std::unique_ptr<llvm::Module>> ModuleOrErr =
-          getOwningLazyBitcodeModule(std::move(*BCBuf), *VMContext);
-      if (!ModuleOrErr) {
-        handleAllErrors(ModuleOrErr.takeError(), [&](ErrorInfoBase &EIB) {
+      if (StringRef(F.Filename).endswith(".a")) {
```

Comment:
Will test if this is required (for now).

Reply:
Nope, none of our stuff requires linking archive formats. I think libm used to. Can be split out from this